benchflow-experiment-manager/application/src/main/java/cloud/benchflow/experimentmanager/models/BenchFlowExperimentModel.java

```diff
@@ -27,6 +27,9 @@
   private BenchFlowExperimentState state;
   private RunningState runningState;
   private TerminatedState terminatedState;
+  private int numTrials;
+  // TODO - this should be part of the DSL
```

Comment:
yes - you are referring to the number of times we should retry to submit the trial to Faban. This number is the number of times we want to run an experiment again if there is some random error during the execution.